pkg/snapshot/overlay.go

```diff
@@ -885,31 +885,17 @@ func (o *snapshotter) identifySnapshotStorageType(id string, info snapshots.Info
 
 	}
 
+	// check overlaybd.commit
```

Comment:
Treat writable snapshot as storageTypeLocalBlock, so that the writable snapshot can be correctly handled when remove.